config/settings.yml

```diff
@@ -410,6 +410,12 @@ REDIS_URL: "redis://localhost:6379/"
 SOLR_COMMIT_WITHIN: 5000
 GEOMONITOR_TOLERANCE: 0.2
 
+# Cloudflare turnstile settings for bot challenge page
+# NOTE: these example keys always pass the challenge; see:
+# https://developers.cloudflare.com/turnstile/troubleshooting/testing/
+TURNSTILE_SITE_KEY: '1x00000000000000000000AA'
```

Comment:
I don't think we should capitalize these.  They aren't constants.  The documentation for ruby config doesn't use capitals either: https://github.com/rubyconfig/config

Reply:
Updated here and in the `shared_configs` PR.